internal/scan-manager/scanners/common/src/main/java/org/wso2/security/tools/scanmanager/scanners/common/config/ScannerConfiguration.java

```diff
@@ -0,0 +1,62 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc., WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ *
+ */
+
+package org.wso2.security.tools.scanmanager.scanners.common.config;
+
+/**
+ * Class to represent the scanner configuration.
+ */
+public abstract class ScannerConfiguration implements Configuration {
+
+    private String scannerClass;
+    private String productPathForZipFileUpload;
+    private String productPathForGitFileClone;
```

Comment:
Do we use Git clone? How can we extend the support (let's say to add downloading from a public HTTP url)